### ISSUE TITLE
Creating a draft of Contributing.md

### ISSUE THREAD

**Issue by marlitas** [phetsims/phet-info#209]
As part of our POSE work we are creating a draft for Contributing.md including a pull request process. This issue will track that work.

**Comment by liammulh** [comment]
TL;DR: phethelp@colorado.edu gets an email when CLA is signed. We need a way to alert devs when someone signs CLA.

Slack convo about CLA:


Liam Mulhall
How do we know when someone has signed it? Does someone get an email?

Liam Mulhall
Oliver, do you know anything about this?

Oliver Nix
Yes - I created this powerform. On completion, a copy is emailed to phethelp@colorado.edu

Oliver Nix
Do we need to add more recipients?

Liam Mulhall
Possibly. If we need open source contributors to sign the CLA, then we need some way of alerting the developers that a contributor has signed the CLA. Ideally we could automate this. Anyway, I will post this info in the GitHub issue where we’re tracking this.

Sam Reid
I saw in this MS repo: https://github.com/microsoft/GenerativeImage2Text

> When you submit a pull request, a CLA bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.

Liam Mulhall
Right, I’ve seen other projects use bots for this.

Sam Reid
https://github.com/cla-assistant/cla-assistant
